lib/eleventhframe.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# similar to frame, but for the eleventh frame
```

Comment:
As you say, similar to frame so perhaps frames 11 & 12 could have inherited from the Frame class which would have kept these additional classes a little dryer.